### ISSUE TITLE
A standardised way of providing API documentation

### ISSUE THREAD

**Issue by bartjkdp** [VNG-Realisatie/nlx#125]
It would be nice to have a generic way of providing API documentation with the API. NLX should retrieve this documentation accordingly and automatically generate a documentation UI. We could support

- OpenAPI (REST)
- Protofiles (gRPC)
- WSDL (SOAP/XML)

Depends on #124
